src/main/java/org/apache/fineract/api/OperationsDetailedApi.java

```diff
@@ -295,10 +309,53 @@ public Map<String, String> filterTransactionRequests(
         return null;
     }
 
+    /*@GetMapping("/test")
```

Comment:
Remove these comments or convert it to a test.

Reply:
I created this endpoitn for testing the new JPQL query. Will remove this.